src/polymer/expression-scanner.ts

```diff
@@ -492,8 +492,9 @@ export function parseExpressionInJsStringLiteral(
     warnings
   };
   const sourceRangeForLiteral = document.sourceRangeForNode(stringLiteral)!;
+  const binaryExpressionString = astValue.binaryExpressionToString(stringLiteral);
 
-  if (!babel.isLiteral(stringLiteral)) {
+  if (binaryExpressionString === null && !babel.isLiteral(stringLiteral)) {
```

Comment:
we are explicit here instead of just a falsy check because i suppose it could be an empty string